README.md

```diff
@@ -1,6 +1,6 @@
 # Google Student Training in Engineering Program
 
-This repo contains the projects you'll work on for the first half of STEP.
+This repo contains Alicja's portfolio and STEP projects.
```

Comment:
nit pick: for later projects, you might want to split a PR into pieces that belong to together. For instance, changing the README should be separate from code changes.